Sources/NIO/CloseOnErrorHandler.swift

```diff
@@ -0,0 +1,23 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors

```

Comment:
2019 ?

Reply:
Oops, I wondered when I copy/pasted 🙈  Fixed!
Do you want me to squash into one commit since only the first matches the template?

Reply:
Squashing the commits would be great!

Reply:
Done!